src/logstash-filters/snippets/doppler.conf

```diff
@@ -0,0 +1,71 @@
+# Parse Cloud Foundry logs from doppler (via https://github.com/SpringerPE/firehose-to-syslog)
+
+json {
+    source => 'syslog_message'
+    add_tag => [ 'cloudfoundry_doppler' ]
+}
+
+if "cloudfoundry_doppler" in [tags] {
```

Comment:
Hmm - turns out than the `json` parser does "tag on failure" - to `_jsonparsefailure`; there is just no way to configure what the failure tag is.  Let me see if I can use that fact to implement our "fail/filter/section" pattern and clean this up...